inf-ruby.el

```diff
@@ -109,11 +109,12 @@ Currently only affects Rails and Hanami consoles."
           (const ask :tag "Ask the user")
           (string :tag "Environment name")))
 
-(defconst inf-ruby-prompt-format
+(defvar inf-ruby-prompt-format
   (concat
    (mapconcat
     #'identity
     '("\\(^%s> *\\)"                      ; Simple
+      "\\(^chef ([\\.[:digit:]]+)> *\\)"  ; Chef Shell: e.g. "chef (13.2.20)> "
```

Comment:
How does the Chef console prompt look in two cases:

* Right after start
* When you input `2 +` and press `RET`

?